src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.tsx

```diff
@@ -46,6 +46,55 @@ interface CheckedProp {
 
 export type MultiSelectStatus = FilterInputStatus & {};
 
+type AriaOptionsAvailableProps =
+  | {
+      ariaOptionsAvailableText?: never;
+      ariaOptionsAvailableTextFunction: (length: number) => string;
+    }
+  | {
+      /** Text for screen reader to read, after labelText/chipText, when selected option is removed from chip list.
+       * E.g 'removed' as prop value would result in '{option} removed' being read by screen reader upon removal.
+       */
+      ariaOptionsAvailableText: string;
```

Comment:
This documentation is copy-pasted from ariaOptionChipRemovedText. Should write a correct one :)

Reply:
Fixed